contents/blog/early-stage-analytics.md

```diff
@@ -0,0 +1,78 @@
+featuredImageType: full
+category: Startups
+tags:
+ - Guides
+---
+
+Building a startup is like trying to find the exit of a dark cave. Sometimes you hear sounds leading you somewhere, but it's mostly dark and hard to understand. In this cave, analytics is like a flashlight. It doesn’t automatically point you to the exit, but it shows you where you're going. You stumble less and spend less time moving in the wrong direction.
+
+![Flashlight](../images/blog/early-stage-analytics/light.png)
+
+Our team works with a lot of high-growth startups (they are [our ideal customers](/handbook/strategy/ideal-customer-persona)). We succeed when they succeed. We know from experience that understanding your users is a vital part of finding early success, and you can't do that without analytics. Here we explain how to use analytics effectively at an early stage.
+
+## Why do analytics matter for startups?
+
+As a simple example, product analytics show you whether your users are actually using your product. Not just "oh yeah, I’ll check it out," but seeing if they are logging in, using the features you developed, and most importantly, returning to use them again. Analytics show what areas they are finding value in, what needs to improve, and what is broken.
+
+From this usage, you can begin creating insights to help you track [key product metrics](/blog/b2b-saas-product-metrics) like traction, engagement, active users, revenue, and feature usage. Without this, you rely heavily on anecdotal evidence. You simply want to see these numbers go up and to the right, there’s not a lot of deep analysis needed here. If the metrics aren’t moving in the right direction, it is a sign to change.
+
+And startups are defined by change. They're all about testing, rapid feedback, and constant improvement. Real usage analytics is the most rapid feedback there is, so starting early is a competitive advantage you shouldn't miss.
+
+For example, we at PostHog use analytics to track OKRs, debug issues, test new features, discover opportunities, and more. To narrow in on OKRs, every team connects them to analytics:
+
+- Team session recordings have "percentage of users per organization viewing recordings"
+- Team product analytics have "top 10 US & EU clients 95th percentile of dashboard load time is under 5 seconds"
+- Team experimentation have "double the number of organizations that use feature flags"
+
+These metrics provide clear, rapid feedback on whether what we are building is valuable and making a difference. Without this, we would spend more time directionless and building features that might not matter to our users.
+
+## The 80/20 of early-stage startup analytics
+
+As much as we value analytics, we believe the best way to create a successful startup is to spend time building a better product, not analytics analysis. To ensure you have time to build, here's what we recommend to get the most benefit from analytics in the shortest time.
```

Comment:
"analytics analysis" is brain melter. Need an alternative.. "not obsessing over data"?